### DISCUSSION TITLE
Phalcon mvc micro app stopped working.

### DISCUSSION THREAD

**Discussion by JohnSereno** [phalcon/phalcon#200] [Q&A]
Several years ago I wrote a restful app using phalcon\mvc\micro. 
when I upgraded PHP to version 7 on my cloud server. It stopped working. I read somewhere that Phalcon would not work with the new version of PHP. I have not done anything in Phalcon for several years. What is the best way for me to get this app working again? Short of rewriting the app.

**Comment by Jeckerson** [comment]
Hello, please specify more details, especially web server logs.

If you need more real time response, please use our official discord server - https://phalcon.io/discord
